values.yaml

```diff
@@ -3,6 +3,10 @@
 
 # Available parameters and their default values for the Vault chart.
 
+# Set cluster name instead of having it auto-generated.
+# examples may include: "vault1-gke-uswest", "vault2-eks-useast", etc.
+cluster_name: ""
```

Comment:
This is a Vault specific parameter and should not be at the root level but instead should be in `server.ha` (it's only used for HA).